src/refactorings/change-signature/change-signature.ts

```diff
@@ -0,0 +1,198 @@
+import { Editor, ErrorReason, SelectedPosition } from "../../editor/editor";
+import { Selection } from "../../editor/selection";
+import * as t from "../../ast";
+import { Path } from "../../editor/path";
+
+let toModifyNode: t.FunctionDeclaration;
+
+export async function changeSignature(editor: Editor) {
+  const params =
+    toModifyNode?.params.map((p, index) => {
+      return {
+        label: getParamName(p),
+        value: {
+          startAt: index,
+          endAt: index
+        }
+      };
+    }) || [];
+
+  await editor.askForPositions(params, async (message) => {
```

Comment:
🍰 `newPositions` may be a better name than `message` here.

Reply:
Fixed at https://github.com/nicoespeon/abracadabra/pull/725/commits/d4d2cd1275c0724182d187e94de0dba685cd6c34